.github/workflows/check_tests.yml

```diff
@@ -98,7 +98,7 @@ jobs:
           SCCACHE_DIR: /home/runner/.cache/sccache
         run: |
           source ${HOME}/.cargo/env
-          cargo test --features=runtime-benchmarks,try-runtime
+          cargo test --release --features=runtime-benchmarks,try-runtime
```

Comment:
why did you bump this test to `--release` optimization? Optimization shouldn't affect any ordering

Reply:
it was to not oom due to the manta-pay tests.
that change is already in the manta branch.